File: Java/2165.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
		Scanner leitor = new Scanner(System.in);
		
		String texto = leitor.nextLine();
		if (texto.length() <= 140) {
			System.out.println("TWEET");
                }
        else {
			System.out.println("MUTE");
        }
    }
}

File: C/2165.c
#include <stdio.h>
int main()
{
	char frase[500];
	scanf("%[^\n\r]", frase);

	if ((strlen(frase) <= 140)) {
	    printf("TWEET\n");
	}
	else {
		printf("MUTE\n");
	}
	return 0;
}